common/src/main/java/com/wynntils/core/net/NetResult.java

```diff
@@ -101,6 +101,7 @@ private void doHandle(Consumer<InputStream> onCompletion, Consumer<Throwable> on
                 });
     }
 
+    @SuppressWarnings("TryFinallyCanBeTryWithResources")
```

Comment:
do you think we need these @magicus

Reply:
![image](https://github.com/Wynntils/Artemis/assets/66840021/2c3f0f14-b95f-4e93-a5df-e588cf875801)

Reply:
Context matters. Afaik we only added the unchecked ones. By default, only we (mostly magicus) do these cleanups (with me reviewing) since this requires a much broader knowledge about the code base, than anyone else has right now. Even if we create and review these PRs, there is a chance we might break something. That's why we are strict reviewing this (and any other PRs).

Reply:
you also added deprecation

Reply:
Yes. Deprecation has a meaning of "dont use this" (also it might be from a supermethod). Unchecked has a meaning of "these raw generics / casts look sketchy but we know what we are doing here". Adding 3 random suppress warning is just litter. Just like we disallowed Nonnull/Nullable annotations since they were not used consistently, this shouldn't be either.

Reply:
I agree; these fills no purpose. This is for a check in IntelliJ that we explicitly do not say will pass.